homeassistant/components/onkyo/const.py

```diff
@@ -0,0 +1,92 @@
+"""Constants for the Onkyo integration."""
+
+DOMAIN = "onkyo"
+
+DEVICE_INTERVIEW_TIMEOUT = 5
+DEVICE_DISCOVERY_TIMEOUT = 5
+
+CONF_RECEIVER_MAX_VOLUME = "receiver_max_volume"
+
+CONF_VOLUME_RESOLUTION = "volume_resolution"
+CONF_VOLUME_RESOLUTION_DEFAULT = 50
+
+OPTION_MAX_VOLUME = "max_volume"
+OPTION_MAX_VOLUME_DEFAULT = 100.0
+
+OPTION_SOURCES = "sources"
+OPTION_SOURCES_DEFAULT = {
+    "tv": "TV",
+    "bd": "Bluray",
+    "game": "Game",
+    "aux1": "Aux1",
+    "video1": "Video 1",
+    "video2": "Video 2",
+    "video3": "Video 3",
+    "video4": "Video 4",
+    "video5": "Video 5",
+    "video6": "Video 6",
+    "video7": "Video 7",
+    "fm": "Radio",
+}
+# This should be kept in sync with strings.json for dynamic options generation
+# to work correctly.
+OPTION_SOURCE_PREFIX = "source."
+OPTION_SOURCES_ALLOWED = [
```

Comment:
This should be an `enum`.

Furthermore I think some of these values are not entirely unique, so it probably needs a little rethinking how to best approach it.

Reply:
Fixed myself. Can be marked resolved.